app/kubernetes/components/datatables/applications-stacks-datatable/applicationsStacksDatatable.html

```diff
@@ -178,7 +178,7 @@
           <td>{{ item.Applications.length }}</td>
           <td>
             <a class="vertical-center" ui-sref="kubernetes.stacks.stack.logs({ namespace: item.ResourcePool, name: item.Name })">
-              <pr-icon icon="'file-text'" size="'md'" title="Logs" feather="true"></pr-icon>
+              <pr-icon icon="'file-text'" size="'md'" title="Logs"></pr-icon>
```

Comment:
EE doesn't have the size prop. which one is correct?

https://github.com/portainer/portainer-ee/pull/2472/files#diff-f28f79bb14461bf2d4822a175f8c8cbf110c88d97b880d5ddfb852cb578df2b2R182

Reply:
removed the size, the default size (like in EE) is better